### PR TITLE
added windows support for chrome

### PR DESCRIPTION
**Pull request by zippoxer**

### CODE REVIEW

File: chrome_windows.go [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+	LPWSTR pDescrOut =  NULL;
+	input.cbData = len;
+	input.pbData = in;
+	CryptUnprotectData(
+		&input,
+		&pDescrOut,
+		NULL,                 // Optional entropy
+		NULL,                 // Reserved
+		NULL,                 // Here, the optional
+							  // prompt structure is not
```
Comment (zellyn) [inline chrome_windows.go]: Weird whitespace

**Comment by zellyn** [comment]
Thanks, btw!

**Comment by zippoxer** [comment]
Np :)

For some reason, github wouldn't accept my whitespace fix, so I removed that comment.

File: chrome_other.go [hunk 2]
```diff
@@ -1,4 +1,5 @@
 // +build !darwin
+// +build !windows
```
Comment (zippoxer) [inline chrome_other.go]: Forgot to add that :P

**Pull request merged**